scripts/common.py

```diff
@@ -418,6 +418,15 @@ def check_prs_for_current_release_are_released(repo):
                 'released before release summary generation.')
 
 
+def convert_to_posixpath(file_path):
+    """Occassionally the Windows style file path just does not work, so we
+    may want to convert it to posix style.
```

Comment:
Add Args and Returns sections to docstring.

Reply:
Updated